apps/forklift/lib/forklift/event/event_handler.ex

```diff
@@ -78,6 +81,27 @@ defmodule Forklift.Event.EventHandler do
       :discard
   end
 
+  def handle_event(%Brook.Event{
+        type: ingestion_update(),
+        data: data,
+        author: author
+      }) do
+    Logger.info("Ingestion: #{data.id} - Received ingestion_update event from #{author}")
+
+    ingestion_update()
+    |> add_event_count(author, data.id)
+
+    Forklift.Ingestions.update(data)
+
+    :ok
+  rescue
+    error ->
+      Logger.error("ingestion_update failed to process. #{inspect(error)}")
+      DeadLetter.process([data.id], nil, data, Atom.to_string(@instance_name), reason: inspect(error))
```

Comment:
First param should be the ingestion's target datasets. 2nd param should be the ingestion ID